src/server.h

```diff
@@ -1706,6 +1706,7 @@ struct redisServer {
     int repl_diskless_load;         /* Slave parse RDB directly from the socket.
                                      * see REPL_DISKLESS_LOAD_* enum */
     int repl_diskless_sync_delay;   /* Delay to start a diskless repl BGSAVE. */
+    int repl_diskless_sync_max_replicas;/* max replicas for diskless repl BGSAVE (will shorten the delay if they all connect). */
```

Comment:
Who will scroll right to read this comment? Not me. To make it visible, break it in the same way as the comments a few lines above.

```suggestion
    int repl_diskless_sync_max_replicas; /* max replicas for diskless repl
                                     * BGSAVE (will shorten the delay if they
                                     * all connect). */
```

I already assume you will disagree. :grin:

Reply:
> and i also think that most people have much wider monitors than they used to

That's true, but some people's eyes are starting to feel irritated and they (me) use an incredibly large font size to compensate. :-) (And/or use multiple narrow editor windows.)